lib/cell.rb

```diff
@@ -0,0 +1,46 @@
+class Cell
+  attr_reader :coordinate,
+              :ship,
+              :render
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = nil
+    @fired_upon = false
+    # @render = "."
+  end
+
+  def empty?
+    @ship == nil
+  end
+
+  def place_ship(new_ship)
+    @ship = new_ship
```

Comment:
@sarahgarlock  future edge case consideration for place_ship method. We wouldn't want someone to be able to overlap ship coordinates -- aka we only want someone to be able to put a ship in the cell IF the cell is nil or empty. So perhaps we say:

`  if empty? 
      @ship == new_ship
    end
    
    we could even add an else statement - "this cell already contains a ship, please choose a new coordinate"
    
    thoughts?

Reply:
Added `if empty?` and else statement